backend/src/packages/surveys/survey.service.ts

```diff
@@ -0,0 +1,55 @@
+import { type Service } from '#libs/types/types.js';
+import { userService } from '#packages/users/users.js';
+
+import { type SurveyGetAllResponseDto } from './libs/types/types.js';
+import { SurveyEntity } from './survey.entity.js';
+import { type SurveyRepository } from './survey.repository.js';
+import {
+  type SurveyGetAllItemResponseDto,
+  type SurveyRequestDto,
+} from './surveys.js';
+
+class SurveyService implements Service {
+  private surveyRepository: SurveyRepository;
+
+  public constructor(surveyRepository: SurveyRepository) {
+    this.surveyRepository = surveyRepository;
+  }
+
+  public find(): ReturnType<Service['find']> {
+    return Promise.resolve(null);
+  }
+
+  public async findAll(): Promise<SurveyGetAllResponseDto> {
+    const items = await this.surveyRepository.findAll();
+
+    return {
+      items: items.map((item) => item.toObject()),
+    };
```

Comment:
Don't implement methods we don't use, return Promise.resolve here, and also remove types related to it like SurveyGetAllResponseDto